go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/aws/aws-sdk-go v1.25.16
 	github.com/briandowns/spinner v1.7.0
 	github.com/fatih/color v1.7.0
+	github.com/fatih/structs v1.1.0
```

Comment:
I believe it's archived not because of its quality but because https://arslan.io/2018/10/09/taking-an-indefinite-sabbatical-from-my-projects/

Also the we are actually using the color library from the same author :)

Reply:
I think what I was trying to actually say is that maybe we can get away with not needing this by simplifying the model a bit so we could just have simple string-string maps. 

Not a big deal though.